### DISCUSSION TITLE
Zabbix when running as container, config datasource in grafana

### DISCUSSION THREAD

**Discussion by Janilassila78** [zabbix-community/helm-zabbix#66] [Q&A] [answered]
We are running zabbix in k8s, we have 4 services 1:server, 1:proxy 1:web 1:webservice
Everything is working fine

But we want to conf zabbix as a datasource in grafana, unfortunately there is not much documantation about it when its running in containers.

Has anyone tried it and got that to work?
what could be the url to create this datasource in grafana?

**Comment by aeciopires** [comment, marked as answer]
Hello @Janilassila78!

This is a same subject commeted in this topic: https://github.com/zabbix-community/helm-zabbix/discussions/27 

You need exposed the Zabbix Frontend, Server and/or Proxy installed in the Kubernetes cluster to allow the access by Grafana installed out of k8s
